app.js

```diff
@@ -470,7 +470,7 @@ d.run(() => {
 						workers: scope.config.wsWorkers,
 						port: scope.config.wsPort,
 						host: '0.0.0.0',
-						wsEngine: 'sc-uws',
+						wsEngine: scope.config.wsEngine || 'sc-uws',
```

Comment:
This config need to be introduced in the configuration files and its default value should be initialized under `helpers/config.js`. Possible values for this configuration option should be documented in README or on our documentation site. 

We should also document it somewhere, that which web socket engine is our recommended one and when to switch to any other and specially which other. 

I don't think we should keep this option open for people to choose what ever websocket engine. We should recommend a few for which we had done our testing.

Reply:
Yes, we should document it somewhere.

I think that we should use `sc-uws` as the default since this is the one which we have done all our testing with and it's the fastest one but we should allow the user to switch to `ws` if all else fails; both `sc-uws` and `ws` are bundled into Lisk core by default and they're fully compatible with one another (both protocol and API).

Reply:
`helpers/config.js` is not just for command line options. Its also used to initialize default values for runtime configurations. Also we need to validate this wsEngine option to allow only `ws` and `sc-uws`

Reply:
@nazarhussain We don't necessarily need to restrict it. If the user wants to use a custom WebSocket server module which they wrote themselves (or a compatible third-party one) then that should be OK too.

OK, I'll move the default `wsEngine` declaration to `helpers/config.js` in case it's null or undefined.